### ISSUE TITLE
Integrate with ITGlue 

### ISSUE THREAD

**Issue by Nickbursey** [DeskDirector/UserVoice#400]
I love that you can now upload URL's to learning centre! But an integration with ITGLUE would be a true game changer.

**Comment by Andrew-Lahikainen** [comment]
@Nickbursey how would you see the integration working? What UX would you expect for the end-user?

**Comment by Nickbursey** [comment]
Hi Andrew

Two elements would make the integration perfect:

1. A link between the subject line of a new ticket request and the learning centre articles, so that as they type a subject it offers articles we've added. Using tags id imagine..

2. For the public IT Glue guide to be presented within a frame inside the portal, rather than opening a new window each time.

Thanks

Nick Bursey
Chief Operations Officer
Tel: +44 (0)203 746 3530
www.ethicalit.net<http://www.ethicalit.net>



________________________________

**Comment by Andrew-Lahikainen** [comment]
@Nickbursey for your second request you'll need to ask IT Glue to allow this. Perhaps they can add a feature which allows you to whitelist domains so it can be embedded in Portal (or they could hard-code the DeskDirector domain in their x-frame-options, but this is less flexible).

**Comment by FluidNets-DamianS** [comment]
There is a trick to make embedding work.  If you set your itglue custom domain to match you dd portal domain it will (or used to work).

Damian C. Stalls
vCIO Director
805.856.1815 Direct

(Please excuse any typos, auto-correct and I don't usually get a long)

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by prysmcorp** [comment]
Yes, we constantly struggle to update learning center links with new articles since that is an additional workflow element. It would be great if articles could be pulled automatically via API. Then as @Nickbursey has said the ability for end-users to have relevant KB articles presented to them as part of a ticket generating process would be phenomenal.

**Comment by iskolares** [comment]
related ticket #39186
